### ISSUE TITLE
Ledger: Confirm IOTA warning message disappears

### ISSUE THREAD

**Issue by f-ben** [iotaledger/trinity-wallet#593]
### Bug description
When setting up the ledger while Trinity Desktop is opened you can get stuck in the "choose ledger account index" screen.

### Environment Specs
- OS version: Windows 10
- Trinity version: Desktop 0.4.1

### Steps To Reproduce
1. Open Trinity Desktop (new or existing account)
2. Open Ledger Live
3. Install IOTA Ledger app but dont open it
4. In Trinity add account and go on as far as you can (until you have to chose the ledger account index)
5. Trinity tells you to open the IOTA Ledger App on your ledger so go ahead and open it

### Expected behaviour
The IOTA Ledger app should display the warning that IOTA is not like other cryptocurrencies. The user should be able to confirm the message and trinity should go on.

### Actual behaviour
The IOTA Ledger app shows the warning message for a few milliseconds and then automatically removes the message and opens the "main menu" of the app. Trinity desktop shows the error "warning unconfirmed: reopen the iota app on your ledger device....".

### Workaround
In Trinity go back one step, open the IOTA Ledger app, confirm the message, in Trinity go forward one step to the Ledger Account Index page.

### Errors
![image](https://user-images.githubusercontent.com/2088471/48306594-acd0db00-e53b-11e8-88cd-9088dd105e9d.png)

### Am I planning to do it myself with a PR?
No

**Comment by Gitulu** [comment]
Hello Guys,

I can confirm this, same version and everything up to date.
Thank you also for this great Workaround.

**Comment by rihardsgravis** [comment]
This is actually a bug on the Ledger IOTA App side. When doing the check request for an open IOTA App, the Ledger resets the user interface and the warning disappears (although it should stay there till accepted). There is no workaround on Trinity side, that is why the error says `Reopen the IOTA app on your Ledger device, accept the informational warning and try again`.

**Comment by f-ben** [comment]
Hey Rihards,
thanks for reply. Reopening the IOTA ledger app doesn´t help. You have to go back one step in trinity or restart trinity completely. If thats not solvable by trinity the message should be extended to something like "Close Trinity, reopen the IOTA app on your ledger device, accept the informational warning and try again"

**Comment by LadaLoveMoney** [comment]
Hi i have same problem, but will not help me "go back one step" What shall I do now?

**Comment by ghost** [comment]
Same problem it's not working even if I close the app

**Comment by rriixx** [comment]
any update on this?

**Comment by hharvin** [comment]
Any update or workaround for this problem yet??

**Comment by LadaLoveMoney** [comment]
No, I lost the iota on ledger, now I'm using trinity and paper backup

**Comment by hharvin** [comment]
Do you think the new nano x will work with IOTA. Got to get my tokens backThanks Hiram
    On Friday, April 12, 2019, 3:31:54 AM EDT, LadaLoveMoney <notifications@github.com> wrote:  
 
 
No, I lost the iota on ledger, now I'm using trinity and paper backup

**Comment by suits-at** [comment]
I just had the same problem, but I got it running.  I could solve it by following these steps:

1. close trinity wallet
2. close ledger live
3. open iota app on ledger nano s
4. accept warning message
5. open trinity wallet again

Hope that helps! :)

**Comment by userreqts** [comment]
I just had the same problem and couldn't get it working even using the tips above.
So I tried it on a different Ledger nano S, and the same thing happened!
It is the first time I have installed the iota app on this device and it didn't even show the warning message - just went straight to the IOTA (underlined)
:(

**Comment by hharvin** [comment]
I got it to finally work. Delete wallet than restore but plug in Ledger and open IOTA wallet and keep scrolling all the way through till the very end than open app on computer and try to restore.
    On Sunday, May 12, 2019, 4:07:33 AM EDT, userreqts <notifications@github.com> wrote:  
 
 
I just had the same problem and couldn't get it working even using the tips above.
So I tried it on a different Ledger nano S, and the same thing happened!
It is the first time I have installed the iota app on this device and it didn't even show the warning message - just went straight to the IOTA (underlined)
:(
